template/cca.ftlh

```diff
@@ -128,17 +135,17 @@
                           margin-right: 0.75rem;
                           margin-bottom: 0.75rem;
                         " href="${serverUrl}/user/show/${memberOf.id}">
-                              ${memberOf.name}<#sep>, </#sep>
-                            </a>
-                          </#if>
-                        </#list>
-                        to contribute to
-                        </a>
-                        <#elseif type=="CCA_DATA_PERMISSION_REMOVED">
-                          Removed permission for users
-                          <#list permissions as memberOf>
-                            <#if (memberOf.id)??>
-                              <a style="
+                                ${memberOf.name}<#sep>, </#sep>
```

Comment:
remove these unnecessary whitespace formatting changes. it does not fall in the usergroup feature context, also the indentations might introduce bugs elsewhere if not tested and looked properly.